core-plugin/src/com/google/gct/idea/debugger/ui/CloudDebugHistoricalSnapshots.java

```diff
@@ -262,9 +262,6 @@ private void onBreakpointsChanged() {
 
     if (breakpointList != null) {
       for (int i = 0; i < breakpointList.size(); i++) {
-        if (breakpointList.get(i).getFinalTime() == null) { // todo: what does this mean?
```

Comment:
why remove this?  It looks like a guard against NPEs , which I think Ben may have seen during testing.